README.md

```diff
@@ -77,6 +77,64 @@ optional argument to logger methods, like this:
 log.error(sentry_skip=True)
 ```
 
+### Sentry Tags
+You can set some or all of key/value pairs of structlog `event_dict` as sentry `tags`:
+
+```python
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,
+        SentryProcessor(level=logging.ERROR, tag_keys=["city", "timezone"]),
+    ],...
+)
+
+log.error("error message", city="Tehran", timezone="UTC+3:30", movie_title="Some title")
+```
+this will report the error and the sentry event will have **city** and **timezone** tags.
+If you want have all event data as tags, create the `SentryProcessor` with `tag_keys="__all__"`.  
+
+
+```python
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,
+        SentryProcessor(level=logging.ERROR, tag_keys="__all__"),
+    ],...
+)
+```
+
+### Skip Extra
+By default `SentryProcessor` will send `event_dict` key/value pairs as extra info to the sentry. 
+Sometimes you may want to skip this, specially when sending the `event_dict` as sentry tags:
+
+```python
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_log_level,
+        SentryProcessor(level=logging.ERROR, as_extra=False, tag_keys="__all__"),
+    ],...
+)
+```
+
+### Logging as JSON
+If you want to configure `structlog` to format the output as **JSON** 
+(maybe for [elk-stack](https://www.elastic.co/elk-stack)) you have to use `SentryJsonProcessor` to prevent
+duplication of an event reported to sentry.
+
+```python
+from structlog_sentry import SentryJsonProcessor
+
+structlog.configure(
+    processors=[
+        structlog.stdlib.add_logger_name,  # required before SentryJsonProcessor()
+        structlog.stdlib.add_log_level,
+        SentryJsonProcessor(level=logging.ERROR, tag_keys="__all__"),
+        structlog.processors.JSONRenderer()
+    ],...
+)
+```
+This processor tells sentry to _ignore_ the logger and captures the events manually. This
```

Comment:
But that's also not ideal, so just remove the comments that it's a temporary workaround as we don't know how temporary it is.

Reply:
Will modify it in the new commit. I wrote it to let a user use this with caution, but I agree that telling what's happening behind is enough.

Reply:
I chose to ignore logger over explicitly disabling logging integration as it allows a user to let sentry capture events of third-party packages as before. Am I wrong? 
I think SentryJsonProcessor is the best we can do for now until sentry_sdk implements some form of event processing/pipeline/formatting before sending to Sentry.

Reply:
Yes, I agree, we probably don't have a better option. I would still remove the temporary workaround comment, because we don't know when it will be possible to remove it. I'd say the comment "to prevent duplication of events" is clear enough.

Reply:
Will remove that part.